amundsen_common/models/dashboard.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+import attr
+from marshmallow_annotations.ext.attrs import AttrsSchema
+
+
+@attr.s(auto_attribs=True, kw_only=True)
+class DashboardSummary:
+    uri: str = attr.ib()
```

Comment:
What is the `uri` format? Is something like `{source}://{cluster}.{group_url}.{url}`?

Reply:
Yes, it is something like:
`<product>_dashboard://<cluster>.<dashboard_group_id>/<dashboard_id>`

e.g: `mode_dashboard://gold.abcdefg/123456789`